draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -2210,13 +2254,13 @@ This section lists recommended suites for hashing to standard elliptic curves.
 A hash-to-curve suite fully specifies the procedure for hashing byte strings
 to points on a specific elliptic curve group.
 {{suites-howto}} describes how to implement a suite.
-Designers specifying a protocol that requires hashing to an elliptic curve
+Designers specifying an application that requires hashing to an elliptic curve
```

Comment:
We use "protocol designers" above and "application designers" here -- why the difference?